dbuf-lsp/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "dbuf-lsp"
+edition = "2021"
+version.workspace = true
+
+[dependencies]
+tokio = {version = "1.43.0", features = ["full"]}
+tower-lsp = "0.20.0"
+dbuf-core = { path = "../dbuf-core"}
```

Comment:
Is it possible to specify dbuf-core as a dependency in a workspace.dependencies table and refer to it here via "workspace = true" instead?